scripts/release_scripts/deploy.py

```diff
@@ -220,12 +221,14 @@ def update_and_check_indexes(app_name):
         'https://console.cloud.google.com/datastore/indexes'
         '?project=%s') % app_name
     gcloud_adapter.update_indexes(INDEX_YAML_PATH, app_name)
-    if not gcloud_adapter.check_all_indexes_are_serving(app_name):
-        common.open_new_tab_in_browser_if_possible(indexes_page_url)
-        raise Exception(
-            'Please wait for all indexes to serve, then run this '
-            'script again to complete the deployment. For details, '
-            'visit the indexes page. Exiting.')
+
+    # Sleep for a minute before checking that all indexes are serving.
+    common.open_new_tab_in_browser_if_possible(indexes_page_url)
+    python_utils.PRINT('Waiting for all indexes to serve...')
+    time.sleep(60)
```

Comment:
Generally, the check takes time if there's a change in index.yaml file (not 100% sure!), and for most of the release I don't think a wait will be required? Considering that should we do the waiting only after checking whether it's required or now?

Reply:
Oh yup -- I confirm that we should do the waiting only if an update is required.

Reply:
Done.

Reply:
Just a note - I am not checking if index.yaml is being modified, that will unnecessarily complicate the script, it is simpler and cleaner to just check if indexes are updated.